simple/README.md

```diff
@@ -13,6 +13,8 @@ python3 -m stats.main
 By default it reads inputs CSVs from the `.data/input` folder and outputs
 a sqlite `datacommons.db` file in the `.data/output` folder.
 
+To enable Data Commons API lookups, set a DC_API_KEY environment variable. See [API documentation](https://docs.datacommons.org/api/rest/v1/getting_started#authentication) to learn more about getting and using API keys.
```

Comment:
can change the link to v2: https://docs.datacommons.org/api/rest/v2/getting_started#authentication
as v1 isn't directly linked from https://docs.datacommons.org/ anymore.